vnext/Desktop.DLL/React.Windows.Desktop.DLL.vcxproj

```diff
@@ -64,21 +61,6 @@
       <GenerateDebugInformation>DebugFull</GenerateDebugInformation>
     </Link>
   </ItemDefinitionGroup>
-  <ItemDefinitionGroup>
-    <ClCompile>
-      <PreprocessorDefinitions>NOJSC;%(PreprocessorDefinitions)</PreprocessorDefinitions>
-    </ClCompile>
-  </ItemDefinitionGroup>
-  <ItemDefinitionGroup Condition="'$(Configuration)'=='Debug'">
-    <ClCompile>
-      <PreprocessorDefinitions>_DEBUG;DEBUG;%(PreprocessorDefinitions)</PreprocessorDefinitions>
-    </ClCompile>
-  </ItemDefinitionGroup>
```

Comment:
Please confirm it's safe to remove these too, i.e. not defined elesewhere .. Sometime we do rely on these flags in code ..

Reply:
These are now redundant. Those flags are implicitly injected by MSBuild/MSVC property sheets.
See logs at https://office.visualstudio.com/ISS/_build/results?buildId=2440053, for Desktop Debug and Release profiles. `_DEBUG` and `NDEBUG` show respectively.

Another evidence is, if you create a new C++ project in VS 2019, it also doesn't explicitly set this flags in the VCXPROJ file anymore.